system/HTTP/CURLRequest.php

```diff
@@ -650,7 +650,7 @@ protected function setCURLOptions(array $curlOptions = [], array $config = [])
         }
 
         // version
-        if (! empty($config['version'])) {
+        if (array_key_exists('version', $config) && $config['version']) {
```

Comment:
right side change to is_string and !== ''?

Reply:
Base documentation float and string